index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/162dfb3961.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+        <!-- Toolbar -->
+        <header class="header">
+            <nav class="header-nav flex">
+                <a href="#" class="logo">
+                    <img src="images/Welcome.png" alt="welcome logo">
+                </a>
+                <a href="#" class="nav-menu">
+                    <img src="images/Union.png" alt="hamburger menu">
+                </a>
+            </nav> 
+        </header>
+
+        <!-- Headline Section -->
+        <section class="headline flex">
+            <h1 class="headline-title">Hi there. I'm Mohamed<br><span>I'm a software developer</span></h1>   
+             <p class="headline-subtitle">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+        
+                <!-- social media icons -->
+                <!-- <a href="#" class="icons">
+                    <img class="icon-img" src="images/social-media-icons.png" alt="Social Media icons">
+                </a> -->
```

Comment:
- [ optional] please delete commented code always when you are done with the project.